src/t_stream.c

```diff
@@ -1475,12 +1542,24 @@ void xrangeGenericCommand(client *c, int rev) {
     }
 }
 
-/* XRANGE key start end [COUNT <n>] */
+/* XRANGE key start end [COUNT <n>]
+ * The 'start' and 'end' IDs are parsed as follows:
+ *   Incomplete 'start' has its sequence set to 0, and 'end' to UINT64_MAX.
+ *   "-" and "+"" mean the minimal and maximal ID values, respectively.
+ *   The "(" prefix means an open (exclusive) range, so XRANGE stream (1-0 (2-0
+ *   will match anything from 1-1 and 1-UINT64_MAX.
+ */
 void xrangeCommand(client *c) {
     xrangeGenericCommand(c,0);
 }
 
-/* XREVRANGE key end start [COUNT <n>] */
+/* XREVRANGE key end start [COUNT <n>]
+ * The 'start' and 'end' IDs are parsed as follows:
+ *   Incomplete 'start' has its sequence set to 0, and 'end' to UINT64_MAX.
+ *   "-" and "+"" mean the minimal and maximal ID values, respectively.
+ *   The "(" prefix means an open (exclusive) range, so XRANGE stream (1-0 (2-0
```

Comment:
```suggestion
 *   The "(" prefix means an open (exclusive) range, so XREVRANGE stream (2-0 (1-0
```

Reply:
did you mean to merge that suggestion you made to yourself?
looking at all this cloned text i now have doubts if my request to put it here was right.
maybe a single line referring to details being in xrangeGenericCommand is enough.
unless the text doc is different for these two (i.e. your suggested change here)

Reply:
The changes to the docs are mainly to deprecate the client-side incr/decr logic. Reverting less verbose commenting, and as both commands are single liners the generic comment should stand out.